src/utils/participantIdentity.ts

```diff
@@ -27,3 +31,24 @@ export class ParticipantIdentity {
     return new ParticipantIdentity(inputString);
   }
 }
+
+export const extendParticipant = (p: Participant | LocalParticipant) => {
+  const parsed = ParticipantIdentity.Parse(p?.identity || '');
+  let participant = p as ParticipantExtended;
+  participant.role = parsed.role;
+  participant.fullName = parsed.partyName;
+  participant.personId = parsed.personId;
+  participant.userId = parsed.userId;
+  participant.isRegistered = !!parsed.userId;
+  return participant as ParticipantExtended;
+};
+export const extendLocalParticipant = (p: LocalParticipant) => {
+  const parsed = ParticipantIdentity.Parse(p?.identity || '');
+  let participant = p as LocalParticipantExtended;
+  participant.role = parsed.role;
+  participant.fullName = parsed.partyName;
```

Comment:
I suggest to split to first name and last name

Reply:
We should discuss where to implement this,
I have some questions:
- Do we keep the fullName variable?
- Do we separate the fullName ourselves to first and last, or do we count on the backend to provide that?
- What do we do when we only have fullName from the backend?
- Should we add prefix ("Dr", "Mr" etc)? What about middle name?